external/java/src/main/java/ch/epfl/dedis/lib/darc/Darc.java

```diff
@@ -0,0 +1,119 @@
+package ch.epfl.dedis.lib.darc;
+
+import ch.epfl.dedis.lib.exception.CothorityCryptoException;
+import ch.epfl.dedis.proto.DarcProto;
+import com.google.protobuf.ByteString;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.util.*;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class Darc {
```

Comment:
This needs some little documentation of what the `Darc` class is. Also for each method, please add a small
```
/**
 * description
 * params:
 */
```